ui/packages/consul-ui/lib/startup/templates/body.html.js

```diff
@@ -42,26 +42,28 @@ ${environment === 'production' ? `{{jsonEncode .}}` : JSON.stringify(config.oper
     "codemirror/mode/xml/xml.js": "${rootURL}assets/codemirror/mode/xml/xml.js"
   }
   </script>
-  <script data-app-name="${appName}" data-${appName}-services src="${rootURL}assets/consul-ui/services.js"></script>
+  <script src="${rootURL}assets/consul-ui/services.js"></script>
+  <script src="${rootURL}assets/consul-ui/routes.js"></script>
 ${
   environment === 'development' || environment === 'staging'
     ? `
-  <script data-app-name="${appName}" data-${appName}-services src="${rootURL}assets/consul-ui/services-debug.js"></script>
-  <script data-app-name="${appName}" data-${appName}-routing src="${rootURL}assets/consul-ui/routes-debug.js"></script>
+  <script src="${rootURL}assets/consul-ui/services-debug.js"></script>
+  <script src="${rootURL}assets/consul-ui/routes-debug.js"></script>
```

Comment:
Removal of all the `data-*` attributes as explained at the top of these PR notes.

Also note, we only load in the `*-debug.js` configuration when we are in a development-like environment. This configuration never gets loaded in production.